Packs/Safewalk/Integrations/SafewalkManagement/SafewalkManagement_test.py

```diff
@@ -0,0 +1,42 @@
+"""Base Integration for Cortex XSOAR - Unit Tests file
+
+Pytest Unit Tests: all funcion names must start with "test_"
+
+More details: https://xsoar.pan.dev/docs/integrations/unit-testing
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+You must add at least a Unit Test function for every XSOAR command
+you are implementing with your integration
+"""
+
+import json
+import io
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+# TODO: REMOVE the following dummy unit test function
```

Comment:
Please add Unit Tests for the integration's functions. See more info on how to here: https://pull-request-440--demisto-content-docs.netlify.app/docs/integrations/unit-testing#write-your-unit-tests